src/main/java/frc/team4909/robot/RobotConstants.java

```diff
@@ -1,11 +1,24 @@
 package frc.team4909.robot;
 
 public class RobotConstants {
+  /* Gamepad */
+  public static final int driverGamepadPort = 0;
+  public static final double driverGamepadSensitivity = 0.6; //Value selected from 2018 code
+  public static final int driverGamepadDeadzone = 0; //Value selected from 2018 code
+
+
   /* Intake */
   public static final double cargoIntakeInSpeed = 1.0;
   public static final double cargoIntakeHoldSpeed = 0;
   public static final double cargoIntakeOutSpeed = -1.0;
 
+
+  /* Linefollow */
+  public static final double fastVelocity = 0.5; //constant velocity given to both motors
+  public static final double slowVelocity = 0.2; //value given to motor when trying to turn
+
+
+
```

Comment:
you split up the intake constants ... also why the excessive whitespace? was that intentional?